src/components/MovieInfo.tsx

```diff
@@ -30,6 +34,7 @@ const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
                 <div className="card-body">
                   <div className="card-title lead text-uppercase">{Title}</div>
                   <p className="card-text">Released on: {Year}</p>
```

Comment:
Same here, let's update this part like this, so that we can best use the space among these elements: 
`<p className="card-text fs-6 text-success h5">Released on: {Year}</p>`